hummingbot/strategy/hanging_orders_tracker.py

```diff
@@ -171,10 +171,12 @@ def update_strategy_orders_with_equivalent_orders(self):
         equivalent_orders = self.equivalent_orders
         orders_to_create = equivalent_orders.union(self.orders_to_be_created).\
             difference(self.strategy_current_hanging_orders)
-        orders_to_cancel = self.strategy_current_hanging_orders.difference(equivalent_orders).\
-            difference(orders_to_create)
+        orders_to_cancel = self.strategy_current_hanging_orders.difference(equivalent_orders)
```

Comment:
@aarmoa Will this cover the case when we have a single accumulated hanging order?